### ISSUE TITLE
Problem with updated kernel

### ISSUE THREAD

**Issue by pfried** [jacobalberty/unifi-docker#47]
This seems to be brokenn right now and as far as i can see also affects this repo:

https://community.ubnt.com/t5/UniFi-Routing-Switching/IMPORTANT-Debian-Ubuntu-users-MUST-READ-Updated-06-21/m-p/1968251#M48264

**Comment by jacobalberty** [comment]
I'm running the fix right now, it was in pr #44 . Did you set the
`JVM_MAX_THREAD_STACK_SIZE` environment variable?

**Comment by jacobalberty** [comment]
I've updated README.md to cover this, it should be updated on the docker hub as well shortly.

**Comment by pfried** [comment]
Had to do an installation yesterday, tried everything on thursday before. Wasted 3h until i figured out that the container was rebuilt during the weekend.

My observations were that it did not start, if I logged into the container and set the environment variable and restarted the service the container crashed each time I tried to setup the Controller. I am not 100% sure if that is related. I need to see if i can test the fixed version

thanks for the quick fix
